public/app/plugins/panel/candlestick/types.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  defaultPanelOptions as defaultPanelOptionsBase,
+  defaultCandlestickColors,
+  PanelOptions,
+  CandlestickColors,
+} from './panelcfg.gen';
+
+export const defaultPanelOptions: Partial<PanelOptions> = {
```

Comment:
I needed to create this in order to have default values for colors and legends. See grafana/schematization-and-as-code-project#56